apps/portal/src/components/common/ProductsSection.js

```diff
@@ -847,9 +858,19 @@ function YearlyDiscount({discount}) {
     }
 }
 
-function ProductPriceSwitch({selectedInterval, setSelectedInterval}) {
+function ProductPriceSwitch({selectedInterval, setSelectedInterval, products}) {
```

Comment:
Products won't be always set, if you search on `ProductPriceSwitch`, it isn't set in the `ChangeProductSection` component. So you'll need to add it there too

Reply:
@SimonBackx hm but it seems `products` is already passed to `ChangeProductSection`, and when I go to “Change plans” I can see the indicator. Or where you thinking about something else?
<img width="1324" alt="image" src="https://github.com/TryGhost/Ghost/assets/11353458/dca663fc-0732-4083-aaa0-d4483476b802">

Reply:
I meant that here it is not passed yet: https://github.com/TryGhost/Ghost/blob/c969dd18a8edd07fc22b6ad6774a4c82dd73460d/apps/portal/src/components/common/ProductsSection.js#L1031-L1034

You'll need to pass the products parameter to the `ProductPriceSwitch` component inside the `ChangeProductSection` linked above